epoi/ops/xformers_attn.py

```diff
@@ -98,6 +98,11 @@ def forward(self, query_layer, key_layer, value_layer, attention_mask, p):
             attn_bias = xformers_ops.fmha.attn_bias.LowerTriangularMask()
             if attention_mask is not None:
                 attn_bias = attn_bias.add_bias(attention_mask)
+                attn_bias = attn_bias.materialize(
```

Comment:
Otherwise,
```
`cutlassB` is not supported because:
    attn_bias type is <class 'xformers.ops.fmha.attn_bias.LowerTriangularMaskWithTensorBias'>
```